src/processes.c

```diff
@@ -2083,10 +2086,12 @@ static int ps_read (void)
 			case 'T': stopped++;  break;
 			case 'W': paging++;   break;
 		}
+		if (ps_get_cmdline (
+		     pid, ps.name, cmdline, sizeof(cmdline)) != NULL) {
```

Comment:
Careful! The `ps_get_cmdline` call in `ps_submit_procstat_entry` uses `NULL` as the name. You're not counting the same thing here. Please verify the other uses of such counters as well.

Reply:
Ah, `ps_get_cmdline` uses the name in case it can't open a /proc/{pid}/cmdline file or get anything of value from there. I've changed it to `NULL`.